templates/build-pod/template.json

```diff
@@ -0,0 +1,146 @@
+{
```

Comment:
The `build-pod` template is generic and used in both the `npm-build-pod` AND the `maven-build-pod`, thus the generic name.